File: Easy/1491-Average Salary Excluding the Minimum and Maximum Salary/Solution.rb
# @param {Integer[]} salary
# @return {Float}
def average(salary)
    min_salary = salary.min
    max_salary = salary.max
    sum_salary = salary.sum

    return 1.0 * (sum_salary - min_salary - max_salary) / (salary.length - 2)
end

File: Easy/1491-Average Salary Excluding the Minimum and Maximum Salary/Solution.rs
impl Solution {
    pub fn average(salary: Vec<i32>) -> f64 {
        let min_salary = salary.iter().min().unwrap();
        let max_salary = salary.iter().max().unwrap();
        let sum_salary = salary.iter().sum::<i32>();

        (sum_salary - min_salary - max_salary) as f64 / (salary.len() - 2) as f64
    }
}
